go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/VividCortex/ewma v1.2.0
-	github.com/aws/aws-sdk-go v1.44.127
+	github.com/aws/aws-sdk-go v1.44.127 // indirect
```

Comment:
Amboy had an SQS implementation, but I removed it because it was only an experiment. The `// indirect` means that some other module depends on the AWS SDK, but Amboy does not directly use it.

Reply:
I removed it and ran `go mod tidy` and it wasn't added back. Does that mean it's not really part of the dependency tree? (and if so why doesn't it remove it without me)

Reply:
`go mod tidy` does different things depending on what version of Go you're running + what version is specified in the go.mod. I wouldn't mess with the go.mod manually since it's meant to be managed by the go binary, unless you know what you're doing.

Reply:
🤷 .
It does look like poplar uses it, though not in a package amboy's using.